wandb/sdk/launch/utils.py

```diff
@@ -794,3 +794,32 @@ def recursive_macro_sub(source: Any, sub_dict: Dict[str, Optional[str]]) -> Any:
         }
     else:
         return source
+
+
+def fetch_and_validate_template_variables(
+    runqueue: Any, fields: dict
+) -> Dict[str, Any]:
+    template_variables = {}
+
+    variable_schemas = {}
+    for tv in runqueue.template_variables:
+        variable_schemas[tv["name"]] = json.loads(tv["schema"])
+
+    for field in fields:
+        key, val = field.split("=")
```

Comment:
This won't work if I have 0 or more than 1 `=` in the value I pass. Can we add a check on the length/type of the return value of `=` here and add some test cases? Just want to make sure users get a clear message if they screw that up.